libqtile/widget/battery.py

```diff
@@ -209,6 +209,8 @@ def __init__(self, **config):
 
         configurable.Configurable.__init__(self, **config)
         self.add_defaults(_LinuxBattery.defaults)
+        if isinstance(self.battery, int):
+            self.battery = "BAT" + str(self.battery)
```

Comment:
Maybe do `"BAT{}".format(self.battery)`. It would be good to also update the docs to say that the battery can be an integer.